scipy/spatial/transform/rotation.py

```diff
@@ -2,7 +2,46 @@
 
 import numpy as np
 import scipy.linalg
-import warnings
+import re
+
+
+_axis_to_ind = {'x': 0, 'y': 1, 'z': 2}
+
+
+def _make_elementary_quat(axis, angles):
```

Comment:
I'm neutral about using underscores here or not using (in all similar places). After all we still can control what to import from `rotation`.